scss/settings/_settings.scss

```diff
@@ -95,12 +95,10 @@ $global-menu-nested-margin: 1rem;
 $global-text-direction: ltr;
 $global-flexbox: true;
 $global-prototype-breakpoints: false;
-$global-button-cursor: pointer;
+$global-button-cursor: auto;
```

Comment:
`_settings.scss` should not be in this PR.

Reply:
Right. But there are also outdated / wrong settings ;-)

Reply:
Also check the min css files and how the order of rules is correct then.